From f1a178c55520d3551bb87d1b989e285e64062eca Mon Sep 17 00:00:00 2001
From: Sean Mackrory <mackrorysd@apache.org>
Date: Mon, 13 Feb 2017 16:24:05 -0700
Subject: [PATCH 2276/2748] HADOOP-14079. s3guard: fix breaking link in
 s3guard.md. Contributed by Mingliang Liu.

(cherry picked from commit 2c3f575c4471e16de6dacf1952acdcb0f872c0cf)

Change-Id: I9be033b444901aa92654283dfc0092ae819c894d
---
 .../src/site/markdown/tools/hadoop-aws/s3guard.md  |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/s3guard.md b/hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/s3guard.md
index dac3d19..aa9fc1b 100644
--- a/hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/s3guard.md
+++ b/hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/s3guard.md
@@ -248,7 +248,7 @@ hadoop s3a init -m URI ( -e ENDPOINT | s3a://BUCKET )
 Creates and initializes an empty metadata store.
 
 A DynamoDB metadata store can be initialized with additional parameters
-pertaining to (provisioned throughput)[http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.ProvisionedThroughput.html]:
+pertaining to [Provisioned Throughput](http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.ProvisionedThroughput.html):
 
 ```
 [-w PROVISIONED_WRITES] [-r PROVISIONED_READS]
-- 
1.7.9.5

